joomla/joomla-cms
# [#33777] Allow overwrite files com_media

### How to test
1. Backend login
2. upload a image (com_media)
3. upload the same image again (com_media)
4. You get an error message "Error: File already exists"
5. apply patch
6. access the com_media options
7. read and understand the tooltip of the option "Allow overwriting files"
8. enable the option "Allow overwriting files"
9. make sure that you have the permission to delete files (core.delete) for com_media
10. try to upload a image (that allready exists) again.
### Expected Behavior (after patch)

The old file will delete and the new one is there. (if the user have core.delete for com_media)
Note that if the upload fails in any reasons the file is deleted and can't be restore by Joomla.
### Joomla Feature Tracker

http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_item_id=33777&start=675
